packages/test/test-end-to-end-tests/src/test/summaries.spec.ts

```diff
@@ -505,6 +515,47 @@ describeCompat("Summaries", "NoCompat", (getTestObjectProvider, apis) => {
 		assert.strictEqual(sharedString1.getLength(), 203);
 		assert.strictEqual(sharedString2.getLength(), 203);
 	});
+
+	itExpects(
+		"attempt last summary if parent container disconnects",
+		[
+			{
+				eventName: "fluid:telemetry:Summarizer:Running:Summarize_Op",
+				summarizeReason: "lastSummary",
+			},
+		],
+		async function () {
+			const container = await createContainer(provider, {});
+			const { summarizer } = await createSummarizer(provider, container, {
+				runtimeOptions: {
+					summaryOptions: {
+						summaryConfigOverrides: {
+							state: "enabled",
+							minOpsForLastSummaryAttempt: 1, // This is the key setting
+							maxTime: 60 * 1000,
+							maxOps: 1000,
+							minIdleTime: 60 * 1000,
+							maxIdleTime: 60 * 1000,
+							runtimeOpWeight: 1,
+							nonRuntimeOpWeight: 1,
+							initialSummarizerDelayMs: 0,
+							maxAckWaitTime: 60 * 1000,
+							maxOpsSinceLastSummary: 1000,
+						},
+					},
+				},
+			});
+			const containerRuntime = (summarizer as any).runtime as ContainerRuntime;
+			const submitSumarySpy = sandbox.spy(containerRuntime, "submitSummary");
+
+			const prom = summarizer.run("test");
```

Comment:
Rename to something more meaningful? Something like: `summarizerRunningPromise`?
Even though it doesn't matter in this test, but it creates a possibility of polluting future code when devs copy paste tests/code.